app/Keycloak/Collection/ClientCollection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Collection;
+
+use App\Keycloak\Dto\Client;
+use Ramsey\Collection\AbstractCollection;
```

Comment:
Since we are in a laravel codebase, doesn't it make more sense to use `Illuminate\Support\Collection`?

Reply:
Same goes for all the Collections used